src/resampler.rs

```
@@ -116,6 +116,17 @@ impl State {
     pub fn get_output_latency(&self) -> usize {
         unsafe { speex_resampler_get_output_latency(self.st) as usize }
     }
+
+    pub fn set_quality(&self, quality: usize) -> usize {
+        unsafe { speex_resampler_set_quality(self.st, quality as i32) as usize}
```

If you want to return an error you should map it to the provided enum.